release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2387,6 +2387,7 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=18786
 
 * Installing {product-title} (OCP) version 4.6 on {rh-virtualization-first} requires RHV version 4.4. If you are running an earlier version of OCP on RHV 4.3, do not update it to OCP version 4.6. Red Hat has not tested running OCP version 4.6 on RHV version 4.3 and does not support this combination. For additional information about tested integrations, see link:https://access.redhat.com/articles/4763741[OpenShift Container Platform 4.x Tested Integrations (for x86_x64)]. 
 
+* Currently, an OVN-Kube Node race occasionally leads to an invalid pod IP. This may result in some users being unable to upgrade from {product-title} 4.7.17 to 4.8.0. As a temporary workaround, users can run a forceful `oc delete pod openshift-ingress_router-default-9b58d8984-st7c8` to fix the issue and successfully upgrade from 4.7.17 to 4.8.0. For additional information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1974403[*BZ#1974403*]. 
```

Comment:
`openshift-ingress_router-default-9b58d8984-st7c8` is a specific pod.  Other clusters will have different names for their router pods.  And my understanding of the bug is that the invalid pod IPs could happen for any pods, not just router pods.  Probably best to get the bug assignee to suggest mitigation.  Perhaps there is a query that can be run to list impacted pods or something?

Reply:
I would just say `OVN-Kubernetes Race` Rather than `OVN-Kube Node race`

Reply:
`invalid pod IP` -> `stale pod IP`

Reply:
Let me see if I can come up with a query that directly exposes the pods with stale IPs